test/index.js

```diff
@@ -116,7 +115,7 @@ test('Grunt-localizr', function (t) {
         });
     });
     t.on('end', function() {
-        process.chdir(original);
+        process.chdir(path.join(__dirname, '..'));
```

Comment:
This isn't the same thing as before -- the original was more correct. This makes huge assumptions about what the cwd should be.